src/menu.cpp

```diff
@@ -0,0 +1,13 @@
+#include "menu.hpp"
+
+Menu::Menu() {
+	font.loadFromFile("/usr/share/fonts/truetype/freefont/FreeSans.ttf");
+	pongText.setFont(font);
+	pongText.setString("PONG");
+	pongText.setCharacterSize(50);
+	pongText.setPosition(280, 180);
+	pongText.setColor(sf::Color::White);
+}
+void Menu::refresh(){
```

Comment:
Nothing here, remove it! [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle pls.